style.css

```diff
@@ -0,0 +1,28 @@
+#title{
+    text-align: center;
+ }
+ 
+ 
+ #image{
+    max-width: 100% ;
+    height: auto;
+    display: block;
+ 
+
+ }
+ 
```

Comment:
Hello @Kccendana, I noticed that you applied different spacing after your declaration block. Applying the same spacing will make your code clean and more readable. :four_leaf_clover::four_leaf_clover:
You can read more about coding standards [here](https://blog.codacy.com/coding-standards-what-are-they-and-why-do-you-need-them/#:~:text=What%20are%20coding%20standards%3F,to%20build%20highly%20functional%20code.).

Reply:
The review message you provided is generally professional, but it could benefit from a few minor adjustments. Here's a revised version:

"Hello @Kccendana, I noticed that there are inconsistent spacing patterns in your declaration blocks. To enhance code cleanliness and readability, it's recommended to apply consistent spacing throughout. 🍀🍀 You may find it helpful to refer to coding standards for further guidance. You can read more about coding standards [here](https://blog.codacy.com/coding-standards-what-are-they-and-why-do-you-need-them/#:~:text=What%20are%20coding%20standards%3F,to%20build%20highly%20functional%20code.). Thank you for your efforts!" 👍 

In this version, the feedback remains constructive and polite. It emphasizes the importance of consistency in spacing, provides a suggestion, and offers a resource for additional learning. 💯